releasenotes/notes/add-elide-swaps-b0a4c373c9af1efd.yaml

```diff
@@ -0,0 +1,36 @@
+---
+features:
+  - |
+    Added a new optimization transpiler pass, :class:`~.ElidePermutations`,
+    which is designed to run prior to the :ref:`layout_stage` and will
+    optimize away any :class:`~.SwapGate`\s in a circuit by permuting virtual
```

Comment:
Should you also mention `PermutationGate`s? Also, should we mention in the release notes that this pass will run in optimization level 3 by default?

Reply:
Yeah, I probably should. I wrote this originally when the pass only elided swaps.